explorer/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21.0
 require (
 	github.com/golangci/golangci-lint v1.54.2
 	github.com/hashicorp/go-plugin v1.5.1
-	github.com/ignite/cli v0.27.1
+	github.com/ignite/cli v0.27.2
```

Comment:
It would be great if the other app dependencies are not updated within this PR 🙏